src/providers/mssql/qgsmssqlconnection.cpp

```diff
@@ -463,6 +473,105 @@ QList<QgsVectorDataProvider::NativeType> QgsMssqlConnection::nativeTypes()
          ;
 }
 
+QStringList QgsMssqlConnection::excludedSchemasList( const QString &connName )
+{
+  QgsSettings settings;
+  QString databaseName = settings.value( QStringLiteral( "/MSSQL/connections/" ) + connName + "/database" ).toString();
+
+  return excludedSchemasList( connName, databaseName );
+}
+
+QStringList QgsMssqlConnection::excludedSchemasList( const QString &connName, const QString &database )
+{
+  QgsSettings settings;
+  bool schemaFilteringEnabled = settings.value( QStringLiteral( "/MSSQL/connections/" ) + connName + "/schemasFiltering" ).toBool();
+
+  if ( schemaFilteringEnabled )
+  {
+    QVariant schemaSettingsVariant = settings.value( QStringLiteral( "/MSSQL/connections/" ) + connName + "/excludedSchemas" );
```

Comment:
I think there are quite a few of naked strings in this PR (missing `QStringLiteral`)

Reply:
Indeed, I was thinking with the first `QStringLiteral` it was enough because string was created, but, I checked, I was wrong. I am going to change that...

Reply:
fix here ; https://github.com/qgis/QGIS/pull/40946 (at least for the those added in this PR)